app/src/main/res/menu/login_delete.xml

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
+
```

Comment:
nit - can we remove the whitespace churn if we didn't actually change this file otherwise?

Reply:
We're renaming it, so I just accepted the whitespaces